components/06-examples/machtigen.hbs

```diff
@@ -0,0 +1,78 @@
+{{> '@header' }}
+<main>
+  <div>
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Inloggen</h1>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-md-12">
+            <ul class="nav nav-tabs">
```

Comment:
Graag het `tabs` component gebruiken ipv markup dupliceren. Met de volgende context bereik je hetzelfde:

```
  tabs:
    - label: Voor mezelf
      url: '#'
      modifiers: active
    - label: Voor een ander
      url: '#'
      modifiers: disabled
```

Dan ben je meteen af van die rare whitespace op regel 17 ;-)

Reply:
Daarbij zie ik liever `href="#"`, de URL werkt in dit geval toch niet. Als je er op klikt komt de gebruiker op een 404 uit.

Reply:
Als je het wel wilt laten navigeren moet je iets met de `path` helper doen

Reply:
En: Waarom staat dit in een `.row .col-md-12` ?